batch2/batch/driver/templates/index.html

```diff
@@ -0,0 +1,42 @@
+{% extends "layout.html" %}
+{% block title %}Batch2 Status{% endblock %}
+{% block content %}
+  <h1>Status</h1>
+  <div class="attributes">
+    <div>instance ID: {{ instance_id }}</div>
+    <div>ready cores: {{ ready_cores_mcpu / 1000 }}</div>
+  </div>
+  <h1>Instances</h1>
+  <div class="attributes">
+    <div>pending: {{ n_instances_by_state['pending'] }}</div>
+    <div>active: {{ n_instances_by_state['active'] }}</div>
+    <div>inactive: {{ n_instances_by_state['inactive'] }}</div>
+    <div>deleted: {{ n_instances_by_state['deleted'] }}</div>
+    <div>live free cores: {{ live_free_cores_mcpu / 1000 }}</div>
+  </div>
+  <table class="data-table" id="instances">
+    <thead>
+      <tr>
+        <th>Time Created</th>
```

Comment:
I would have thought name should come first.

Reply:
OK.  I put it before Last Updated.